labeddit/src/routes/coordinator.jsx

```diff
@@ -0,0 +1,15 @@
+export const goToLogin = (navigate) => {
```

Comment:
Não seria interessante passar o useNavigate aqui dentro desse arquivo? pra não ter que importar essa função e também o navigate quando for usar dentro das páginas ou componentes.